droid-hal-device.inc

```diff
@@ -102,6 +101,22 @@ Device specific droid headers for %{rpm_device}.
 Needed by libhybris
 
 ################
+%package tools
+Provides: droid-hal-tools
```

Comment:
We do not need this as hw adaptation repo can make its own decision which package to provide and after this we do not need to have droid-hal-tools at all anymore.